Documentation/docs-mobile/building-apps/build-properties.md

```diff
@@ -1682,21 +1682,6 @@ located.
 The default value is `Resources`. Change this to `res` for the
 Java project structure.
 
-## MonoSymbolArchive
-
-A boolean property that controls
-whether `.mSYM` artifacts are created for later use with
-`mono-symbolicate`, to extract &ldquo;real&rdquo; filename and line
-number information from Release stack traces.
-
-This is True by default for &ldquo;Release&rdquo; apps which have
-debugging symbols enabled:
-[`$(EmbedAssembliesIntoApk)`](#embedassembliesintoapk) is True,
-[`$(DebugSymbols)`](#debugsymbols)
- is True, and
-[`$(Optimize)`](/visualstudio/msbuild/common-msbuild-project-properties)
-is True.
-
```

Comment:
Restore this section of text, but add a NOTE under the `## MonoSymbolArchive` heading that says something like:
```
This was a legacy MSBuild property from Xamarin.Android. Not available in .NET 6+.
```